checkr/src/models/candidate.ts

```diff
@@ -0,0 +1,44 @@
+import { Table, Column, Model, ForeignKey, BelongsTo, Length, IsEmail, isDataType } from 'sequelize-typescript';
+import { User } from './user';
+
+@Table
+export class Candidate extends Model<Candidate> {
+    @Column({ primaryKey: true, autoIncrement: true })
+    candidateId!: number;
+
+    @Length({ min: 3, max: 45 })
+    @Column
+    name!: string;
+
+    @IsEmail
+    @Length({ min: 6, max: 45 })
+    @Column
+    email!: string;
+
+    @Column
+    dob!: Date;
+
+    @Length({ min: 10, max: 10 })
+    @Column
+    phone!: string;
+
+    @Length({ min: 5, max: 6 })
+    @Column
+    zipcode!: string;
+
+    @Column
+    socialSecurity!: string;
+
+    @Column
+    driversLicense!: string;
+
+    @Column
+    createdDate!: Date;
+
+    @ForeignKey(() => User)
+    @Column
+    userId!: number;
+
+    @BelongsTo(() => User)
+    user!: User;
```

Comment:
does this create any user column in candidate table?

Reply:
No, it just gives association methods.